style.css

```diff
@@ -1 +1,175 @@
 /*  Add your styling here */
+body {
+  font-family: "Nunito", sans-serif;
+  color: #333;
+  font-weight: 300;
+  line-height: 1.6;
+  margin: 0;
+  padding: 0;
+  font-size: 20px;
+}
+h1 {
+  color: green;
+  font-family: fantasy;
+  font-weight: bold;
+}
+.grid-container {
+  grid-auto-rows: auto;
+  grid-template-areas:
+    "header header"
+    "main main"
+    "footer footer";
+  padding: 2em;
+  grid-gap: 1em;
+}
+
+header {
+  grid-area: header;
+}
+main {
+  grid-area: main;
+}
+footer {
+  grid-area: footer;
+}
+.navbar {
+  display: none;
+}
+
+.logo {
+  position: relative;
+  display: flex;
+  padding: 0.5em;
+}
+.logo-image {
+  grid-area: logo-image;
+  position: absolute;
+  right: 0%;
+  bottom: 10px;
+  height: 45px;
+  width: 50px;
+}
+
+.logo-text {
+  justify-content: flex-end;
+  font-size: 18px;
+  position: absolute;
+  right: 0%;
+  top: 50px;
+}
+
+.welcome {
+  display: grid;
+  grid-template-areas:
+    "welcome-text welcome-text welcome-text welcome-text welcome-text welcome-text"
+    ". . welcome-image welcome-image  . .";
+}
+
+.welcome-image {
+  grid-area: welcome-image;
+}
+.welcome-text {
+  grid-area: welcome-text;
+}
+/* styling products */
+
+.products {
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: 400px 400px;
+  display: grid;
+  grid-gap: 1em;
+}
+.product-1 {
+  background-image: url("./images/chocolate-cake.jpeg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+.product-2 {
+  background-image: url("./images/vanilla-cake.jpeg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.product-3 {
+  background-image: url("./images/fancy.jpeg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+.product-4 {
```

Comment:
Here we have 4 similar classes with almost identical css rules. We definetly want to avoid it. We want to reuse as much classes as possible for identical shaped items.

If you see that you need to give numbers to your classes like here (e.g. `product-1`, `product-2`, ...), it's a sign that you need to reuse css class.

I.e.

```
.product {
  background-size: cover;
  background-repeat: no-repeat;
  background-position: center;
}
```

But, in this case, it would have been better to use `<img>` rather than a `background-image` because the images are not really a background but an image describing the products on the page. Read more about their differences: https://stackoverflow.com/a/1469139/1189105

Also, with `img` tag you won't need separate classes anymore for sure. Then you can use `object-fit` and `object-position` to place your image in the right location: 
https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_object-fit2